AutoscoperM/AutoscoperMLib/IO.py

```diff
@@ -35,34 +33,6 @@ def loadSegmentation(segmentationNode: slicer.vtkMRMLSegmentationNode, filename:
         return None
 
 
-def generateCameraCalibrationFile(camera: Camera, filename: str):
-    """
```

Comment:
Once this is integrated, let's also make sure to submit a pull request against `BrownBiomechanics/Autoscoper` to add a `deprecated` admonition to https://github.com/BrownBiomechanics/Autoscoper/blob/main/Documentation/file-specifications/camera-calibration.md#overview

Reply:
Opened PR https://github.com/BrownBiomechanics/Autoscoper/pull/307 to update the documentation.